pkg/apis/config/defaults.go

```diff
@@ -44,6 +44,13 @@ const (
 	// DefaultUserContainerName is the default name we give to the container
 	// specified by the user, if `name:` is omitted.
 	DefaultUserContainerName = "user-container"
+
+	// DefaultContainerConcurrency is the default container concurrency. It will be set if ContainerConcurrency is not specified.
+	DefaultContainerConcurrency int64 = 0
+
+	// DefaultMaxRevisionContainerConcurrency is the maximum configurable
+	// container concurrency.
+	DefaultMaxRevisionContainerConcurrency int64 = 1000
```

Comment:
Curious why you moved this as well?

We may want to make this configurable with `"max-container-concurrency"` (cc @mattmoor wdyt?) but as it is, MaxRevisionContainerConcurrency is just a constant that you can't configure. (I'm not sure why it's set to 1000, @vagababov may have more context.)

Reply:
@jonjohnsonjr That's correct. Before it was defined in the v1beta1 package, so I moved it out since it would be used across v1alpha1, v1beta1, and `pkg/apis/serving`. Should we still keep it in v1beta1?

Reply:
Ah actually this can be removed https://github.com/knative/serving/pull/5099#discussion_r313693738

Reply:
Well Autoscaling types, probably might go directly to v1 (or stay forever v1alpha1, since they are internal types).
But It's ok to have constants which are semantic in the general place, though `apis/autoscaling/register.go` might be a better place for it.

@jonjohnsonjr I filed https://github.com/knative/serving/issues/5161